easybuild/easyconfigs/b/BirdNET/BirdNET-20201214-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,104 @@
+easyblock = 'Tarball'
+
+name = 'BirdNET'
+version = '20201214'
+versionsuffix = '-Python-%(pyver)s'
+local_commit = '022a6d5'
+
+homepage = 'https://birdnet.cornell.edu/'
+description = """BirdNET is a research platform that aims at recognizing birds by sound at scale.
+We support various hardware and operating systems such as Arduino microcontrollers, the Raspberry Pi, 
+smartphones, web browsers, workstation PCs, and even cloud services. BirdNET is a citizen science 
+platform as well as an analysis software for extremely large collections of audio.
+BirdNET aims to provide innovative tools for conservationists, biologists, and birders alike."""
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+github_account = 'kahst'
+source_urls = [GITHUB_SOURCE]
+sources = [
+    {
+        'download_filename': '%s.tar.gz' % local_commit,
+        'filename': SOURCE_TAR_GZ,
+    },
+    {
+        'source_urls': ['https://tuc.cloud/index.php/s/m9smX4FkqmJaxLW/'],
+        'download_filename': 'download',
+        'filename': 'BirdNET_Soundscape_Model.pkl',
+        'extract_cmd': 'echo %s',
+    },
+]
+patches = [
+    ('BirdNET-20201214_add-model-path-envvar.patch'),
+    ('BirdNET_Soundscape_Model.pkl', 'model'),
+]
+checksums = [
+    '3a7a5f562bfdd55138603847d206125328f02a25df5e86278d2fb35242bb1c1d',  # BirdNET-20201214.tar.gz
+    '7db0b2fe57da002aff62fe1aecf56b16065c2b699dcbee1d8d7e07a42719b3fe',  # BirdNET_Soundscape_Model.pkl
+    '33cbddca35223eb078ed62e7eb1748930225dfe43a40b9d093ae0d6d18a1344b',  # BirdNET-20201214_add-model-path-envvar.patch
+    '7db0b2fe57da002aff62fe1aecf56b16065c2b699dcbee1d8d7e07a42719b3fe',  # BirdNET_Soundscape_Model.pkl
+]
+
+dependencies = [
+    ('FFmpeg', '4.2.1'),
+    ('libsndfile', '1.0.28'),
+    ('Python', '3.7.4'),
+    ('libgpuarray', '0.7.6', '-Python-%(pyver)s'),
+    ('numba', '0.47.0', '-Python-%(pyver)s'),
+    ('scikit-learn', '0.22.2.post1', '-Python-%(pyver)s'),
+    ('SciPy-bundle', '2019.10', '-Python-%(pyver)s'),
+    ('Theano', '1.0.4', '-Python-%(pyver)s'),
+]
+
+postinstallcmds = [
+    'chmod +x %(installdir)s/analyze.py',
```

Comment:
Not fully sure if postinstallcmds is handled in Tarball. Check the log and installation if that is done.

Reply:
Yep, that actually worked:
```
== 2021-04-20 15:47:41,153 easyblock.py:3138 INFO Running method post_install_step part of step postproc
== 2021-04-20 15:47:41,154 run.py:225 INFO running cmd: chmod +x /home/user/easybuildinstall/software/BirdNET/20201214-fosscuda-2019b/analyze.py 
== 2021-04-20 15:47:41,179 run.py:542 INFO cmd "chmod +x /home/user/easybuildinstall/software/BirdNET/20201214-fosscuda-2019b/analyze.py" exited with exit code 0 and output:
```
```
$ ls -l $EBROOTBIRDNET 
total 13564
-rwxr-xr-x. 1 user user    10046 20 apr 15:34 analyze.py
-rw-r--r--. 1 user user     9968 14 dec 13:03 analyze.py.orig

```